client_it_test.go

```diff
@@ -321,3 +321,112 @@ func TestClusterReconnection_ReconnectModeOff(t *testing.T) {
 	time.Sleep(100 * time.Millisecond)
 	assert.Equal(t, false, c.Running())
 }
+
+func TestClient_GetDistributedObjects(t *testing.T) {
+	const (
+		testMapName = "get-distributed-objects-map"
+		testSetName = "get-distributed-objects-set"
+	)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 40*time.Second)
+	defer cancel()
+
+	cls := it.StartNewClusterWithOptions("go-cli-test-cluster", 15701, 1)
+	time.Sleep(2 * time.Second)
+	defer cls.Shutdown()
+	config := cls.DefaultConfig()
+
+	c, err := hz.StartNewClientWithConfig(ctx, config)
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = c.GetMap(ctx, testMapName)
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = c.GetSet(ctx, testSetName)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err = c.Shutdown(ctx); err != nil {
+		t.Fatal(err)
+	}
+
+	c, err = hz.StartNewClientWithConfig(ctx, config)
+	if err != nil {
+		t.Fatal(err)
+	}
+	objects, err := c.GetDistributedObjects(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	assert.Equal(t, 2, len(objects))
+	hzMap := objects[testMapName]
+	hzSet := objects[testSetName]
+	assert.NotNil(t, hzMap)
+	assert.NotNil(t, hzSet)
+	assert.IsType(t, new(hz.Map), hzMap)
```

Comment:
Doesn't `assert.IsType(t, *hz.Map, hzMap)` work?

Reply:
It expects not type but expression to be passed. So no, it does not work.